test/dates/io.jl

```diff
@@ -323,15 +323,14 @@ f = "ymd"
 dr = ["2000-01-01", "2000-01-02", "2000-01-03", "2000-01-04", "2000-01-05",
       "2000-01-06", "2000-01-07", "2000-01-08", "2000-01-09", "2000-01-10"]
 dr2 = [Dates.Date(2000) : Dates.Date(2000, 1, 10);]
-@test Dates.Date(dr) == dr2
-@test Dates.Date(dr, "yyyy-mm-dd") == dr2
+@test Dates.Date.(dr) == dr2
+@test Dates.Date.(dr, "yyyy-mm-dd") == dr2
```

Comment:
Changed these `Date` and `DateTime` tests to use the `dateformat` string macro. Thanks! :)